pipelines/template-build/template-build.yaml

```diff
@@ -270,9 +270,9 @@ spec:
       - name: pipeline-run-name
         value: "$(context.pipelineRun.name)"
       - name: git-url
-        value: "$(params.git-url)"
+        value: "$(tasks.clone-repository.results.url)?rev=$(tasks.clone-repository.results.commit)"
       - name: image-url
-        value: $(params.output-image)
+        value: "$(tasks.build-container.results.IMAGE_URL)@$(tasks.build-container.results.IMAGE_DIGEST)"
```

Comment:
I thought the `show-summary` task was meant to always run (which is why it's in `finally`). There are cases where the pipeline skips the build on purpose because it determines that a build is not needed. Do we still want the `show-summary` task to run in such cases? With this change it will not.

Reply:
Correct, that is the side effect of this change. If the `build-container` step is skipped, `show-summary` will be skipped as well.

If we wish to have `show-summary` run always, this needs to be reverted. Obviously the intent with `finally` was to have it run always, that changes now because I wanted to have correct information presented. That's my reasoning at least, I could easily revert this.

WDYT @Michkov?

Reply:
Thinking of removing show-summary task completely